src/renderer_d3d12.cpp

```diff
@@ -844,13 +844,31 @@ namespace bgfx { namespace d3d12
 
 								if (SUCCEEDED(hr))
 								{
-//									debug1->SetEnableGPUBasedValidation(true);
+									//This is the least ugly way to get the windows build that still works past win10 without deprecation warnings or needing app manifests
+									RTL_OSVERSIONINFOW osver;
+									bx::memSet(&osver, 0 , sizeof(RTL_OSVERSIONINFOW));
+									const HMODULE hMod = ::GetModuleHandleW(L"ntdll.dll");
+								    if (hMod) {
+										FARPROC (WINAPI* rtlGetVersionPtr) (PRTL_OSVERSIONINFOW) = reinterpret_cast<FARPROC (WINAPI*)(PRTL_OSVERSIONINFOW)>(::GetProcAddress(hMod, "RtlGetVersion"));
+								        if (rtlGetVersionPtr != nullptr) {
+								            rtlGetVersionPtr(&osver);
```

Comment:
Before you call `rtlGetVersion(&osver)` you need to specify `dwOSVersionInfoSize` size.
https://learn.microsoft.com/en-us/windows-hardware/drivers/ddi/wdm/nf-wdm-rtlgetversion#parameters